pkg/ceph/rgw/objectstore_test.go

```diff
@@ -114,11 +127,17 @@ func TestDeleteStore(t *testing.T) {
 	}
 	context := &Context{context: &clusterd.Context{Executor: executor}, Name: "myobj", ClusterName: "ns"}
 
+	// Delete one of two object stores
```

Comment:
this comment isn't entirely accurate since this func is being called with just 1 store also.

Reply:
right, will update the comment since this was refactored

Reply:
OK let me know when it's been updated.